taichi/rhi/vulkan/vulkan_device.cpp

```diff
@@ -2438,7 +2438,7 @@ void VulkanSurface::create_swap_chain() {
   createInfo.pNext = nullptr;
   createInfo.flags = 0;
   createInfo.surface = surface_;
-  createInfo.minImageCount = capabilities.minImageCount;
+  createInfo.minImageCount = std::min<uint32_t>(capabilities.maxImageCount, 3);
```

Comment:
nit: would you mind adding a comment explaining why `3` is chosen :P? Thanks!

Reply:
@ailzhang https://www.intel.com/content/www/us/en/support/articles/000006930/graphics.html

Reply:
it's a rather common practice actually, you know it if you are a gamer 😎